<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeConfig</class>
 <widget class="QWidget" name="MakeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>362</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&amp;Abort on first error:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_abortOnFirstError</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="kcfg_abortOnFirstError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&amp;Display commands but do not execute them:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_displayOnly</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="kcfg_displayOnly">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>Install as &amp;root:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_installAsRoot</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="kcfg_installAsRoot">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="rootinstallationcommandLabel">
     <property name="text">
      <string>Root installation &amp;command:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_suCommand</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="kcfg_suCommand">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>kdesu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kdesudo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sudo</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Number of simultaneous jobs:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_numberOfJobs</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="kcfg_numberOfJobs">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Make executable:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>makeBinary</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KUrlRequester" name="makeBinary">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Default make &amp;target:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_defaultTarget</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="kcfg_defaultTarget">
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Additional make &amp;options:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_additionalOptions</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="kcfg_additionalOptions">
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Active environment &amp;profile:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_environmentProfile</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="KDevelop::EnvironmentSelectionWidget" name="kcfg_environmentProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KDevelop::EnvironmentConfigureButton" name="configureEnvironment" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KDevelop::EnvironmentSelectionWidget</class>
   <extends>KComboBox</extends>
   <header location="global">util/environmentselectionwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDevelop::EnvironmentConfigureButton</class>
   <extends>QToolButton</extends>
   <header>util/environmentconfigurebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_installAsRoot</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_suCommand</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_installAsRoot</sender>
   <signal>toggled(bool)</signal>
   <receiver>rootinstallationcommandLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
